### PR TITLE
handle resource group file

### PR DESCRIPTION
**Pull request by florianMalbranque**
handle resource group file the same way access control is dealt with

### CODE REVIEW

**Comment by florianMalbranque** [comment]
@cla-bot check

**Comment by florianMalbranque** [comment]
@losipiuk @sopel39 PTAL 🙏

File: charts/trino/values.yaml [hunk 1]
```diff
@@ -96,6 +96,69 @@ accessControl: {}
   #       ]
   #     }
 
+resourceGroups: {}
+  # type: configmap
+  # # Rules file is mounted to /etc/trino/resource-groups
+  # configFile: "resource-groups.json"
+  # resourceGroupsFile:
```
Comment (nineinchnick) [inline charts/trino/values.yaml:103]: What's the use case for specifying `configFile`? Also, why `resourceGroupsFile` is an object, if we don't actually allow/use more than one key?

File: charts/trino/values.yaml [hunk 2]
```diff
@@ -96,6 +96,67 @@ accessControl: {}
   #       ]
   #     }
 
+resourceGroups: {}
+   # Rules file is mounted to /etc/trino/resource-groups/resource-groups.json
```
Comment (nineinchnick) [inline charts/trino/values.yaml:100]: nit: can you align the commented out lines to have the same indentation?

**Review by nineinchnick — APPROVED** [review]

File: charts/trino/values.yaml (hunk 2 above)
Comment (florianMalbranque) [reply charts/trino/values.yaml]: done

File: charts/trino/values.yaml [hunk 3]
```diff
@@ -96,6 +96,66 @@ accessControl: {}
   #       ]
   #     }
 
+resourceGroups: {}
+  # # Resource groups file is mounted to /etc/trino/resource-groups/resource-groups.json
```
Comment (losipiuk) [inline charts/trino/values.yaml:100]: nit: probably you can use smaller example

Comment (florianMalbranque) [reply charts/trino/values.yaml:100]: This example matches the above access control example in terms of users and groups while being smaller than [Trino's documentation example](https://trino.io/docs/current/admin/resource-groups.html), to me it's appropriate

**Comment by losipiuk** [comment]
Please squash commits

**Pull request merged**